backend/lib/rpc-service/utils.js

```diff
@@ -14,6 +18,7 @@ const decodeCborInBase64 = (data) => {
  */
 const rawEventEntriesToEvent = (rawEventEntries) => {
   // Each event is defined by a list of event entries which will parsed into a typed event
+  /** @type {Record<string, any>} */
```

Comment:
We do not yet know the event type at this point, which is why I used any as a type declaration here.

Reply:
Let's use `unknown` instead of `any` please.

```suggestion
  /** @type {Record<string, unknown>} */
```